tests/app.test.ts

```diff
@@ -103,4 +103,53 @@ describe('Application Integration Tests', () => {
                 expect(e.response.status).toBe(400);
             });
     });
+
+    it('uploads a manuscript file', async () => {
+        const body = new FormData();
+
+        const loginResponse = await axios.post(
+            'http://localhost:3000/graphql',
+            {
+                query: `
+                    mutation StartSubmission($articleType: String!) {
+                        startSubmission(articleType: $articleType) {
+                            id
+                        }
+                    }
+                `,
+                variables: {
+                    articleType: 'researchArticle',
+                },
+            },
+            { headers: { Authorization: `Bearer ${jwtToken}` } },
+        );
+
+
+        const id = loginResponse.data.data.startSubmission.id;
+        const query = `mutation UploadManuscript($id: ID!, $file: Upload!, $fileSize: Int!) {
+            uploadManuscript(id: $id, file: $file, fileSize: $fileSize) {
+                id
+            }
+        }`;
+
+        const operations = {
+            query,
+            variables: {
+                id,
+                file: null,
+                fileSize: 2,
+            },
+        };
+
+        body.append('operations', JSON.stringify(operations));
```

Comment:
I don't think its a good idea to tie front end code for the sake of an integration test. The current test uses curl in 5 lines and matches the spec.

Also we could be having another graphql server querying the endpoint

Reply:
We had a conversation about this in the office yesterday - maybe we can catch up and talk about this. I think its worth us understanding the value of this test ....  if this test fails - it might not mean there is an actual failure and if it passes it might mean it doesn't actually work - so what is the value in the test?

Reply:
I don't follow. It's using minio (https://github.com/minio/minio) to simulate s3 so it's a real blob storage running in a container. This test is actually uploading a tiny file. If the operation fails the test will fail.